functional_tests/steps/bundles.py

```diff
@@ -53,3 +54,30 @@ def the_user_opens_page_chooser(context: Context) -> None:
 def the_locale_column_is_displayed(context: Context) -> None:
     modal = context.page.locator(".modal-body")
     modal.get_by_role("columnheader", name="Locale").is_visible()
+
+
+@step("the user can add Bundles created by user")
+def a_bundle_with_name_item_name_has_been_created_by_username(context: Context) -> None:
+    context.bundle = BundleFactory(created_by=context.user_data["user"])
+
+
+@step("Bundle has the creator removed")
+def created_by_has_been_deleted(context: Context) -> None:
+    context.bundle.created_by = None
+    context.bundle.save(update_fields=["created_by"])
+
+
+@step('a "{user_type}" logs into the admin site with username "{username}"')
+def user_logs_in_with_username(context: Context, user_type: str, username: str) -> None:
+    context.user_data = create_user(user_type)
+    context.page.goto(f"{context.base_url}/admin/login/")
+    context.page.get_by_placeholder("Enter your username").fill(context.user_data[username])
+    context.page.get_by_placeholder("Enter password").fill(context.user_data["password"])
+    context.page.get_by_role("button", name="Sign in").click()
```

Comment:
This step is not being used, and is already implemented in `cms_login.py`, albeit without the username parameter:

https://github.com/ONSdigital/dis-wagtail/blob/c4cd24cfe3e57d124d804cef51befdcf911e95f1/functional_tests/steps/cms_login.py#L32-L33